creme/assistants/tests/test_memo.py

```diff
@@ -75,9 +75,11 @@ def test_edit(self):
                         )
 
         # ---
-        content += """: 
-I add a long text in order to obtain a content that 
-will be truncate by str() method"""
+        content += "\n".join([
+            ": ",
+            "I add a long text in order to obtain a content that ",
+            "will be truncate by str() method",
+        ])
```

Comment:
context += (
     ' : \n'
     'I add a long text in order to obtain a content that \n'
     'will be truncate by str() method'
)